pom.xml

```diff
@@ -10,7 +10,7 @@
 
   <groupId>io.jenkins.plugins</groupId>
   <artifactId>environment-dashboard</artifactId>
-  <version>1.1.9-SNAPSHOT</version>
+  <version>1.1.10-SNAPSHOT</version>
```

Comment:
Revert this change as the next version is in fact 1.1.9 occording to tags

Reply:
Done